setup.py

```diff
@@ -37,6 +37,8 @@ def run(self):
             "License :: OSI Approved :: Apache Software License",
             "Programming Language :: Python :: 3.9",
```

Comment:
not sure when 3.8/3.7 was removed, but technically the setuptools still supports 3.7/3.8?

Reply:
Good point, but I'm not sure that we want to declare support for 3.8 if we're not running unit tests for it.